daal4py/sklearn/neighbors/tests/test_kneighbors.py

```diff
@@ -0,0 +1,105 @@
+#
+#*******************************************************************************
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#******************************************************************************/
+
+import pytest
+import numpy as np
+
+from daal4py.sklearn.neighbors import KNeighborsClassifier
+from sklearn.model_selection import train_test_split
+from sklearn.datasets import make_classification
+
+DISTANCES = ['minkowski']
+ALGORITHMS = ['brute', 'kd_tree', 'auto']
+WEIGHTS = ['uniform', 'distance']
+KS = [1, 3, 7, 25]
+
+
+def make_dataset(n_samples=256, n_features=5, n_classes=2, random_state=777):
+    x, y = make_classification(n_samples=n_samples, n_features=n_features, n_classes=n_classes, random_state=random_state)
+    return train_test_split(x, y, random_state=777, test_size=0.5, shuffle=True)
```

Comment:
1. random_state=random_state
2. may be it makes sense to add parameters test_size=0.5, shuffle=True to the function for future reuse it in different cases

Reply:
Corrected